pkg/scheduler/test_utils/jobs_fake/jobs.go

```diff
@@ -130,6 +130,7 @@ func BuildJobInfo(
 			},
 		},
 	}
+	_ = result.GetActiveAllocatedTasksCount()
```

Comment:
Does it have any side effect?

Reply:
OK, usually we are supposed to use NewPodInfo which does it by adding the tasks one by one